posts/2024/esds-annual-event-recap.md

```diff
@@ -0,0 +1,63 @@
+---
+author: Katelyn FitzGerald
+date: 2024-02-?
+tags: tutorial hackathon events
+---
+
+# 2024 ESDS Annual Event Recap
+
+The [ESDS Events Working Group](https://ncar.github.io/esds/about/#events-working-group)
+hosted the 2024 ESDS Annual Event January 18-19th in a hybrid format at the NSF NCAR
+Mesa Lab and online.
+
+This blog post shares a brief overview of the event followed by resources and takeaways
+from the event.
+
+## Overview
+
+The two day event consisted of a series of tutorials, office hours, and collaborative
+work sessions as well as a talk from our invited speaker to kick off the event. You
+can find the [full agenda here](https://docs.google.com/spreadsheets/d/1XNZW8NWoF2LaQ4mlo8c06lNnNVJ0ndTG_DfFH8KsatQ/view).
+
+Something about the purpose of the event...
```

Comment:
- Provide access to introductory and advanced Earth system data science focused tutorials
- Encourage collaboration and open development
- Build a welcoming and inclusive community